apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/grpc/RetriableStargateBridgeTest.java

```diff
@@ -37,191 +37,141 @@
-import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
+@TestProfile(RetriableStargateBridgeTest.Profile.class)
 class RetriableStargateBridgeTest extends BridgeTest {
 
-  @Retriable @Inject RetriableStargateBridge bridge;
-
-  @Nested
-  @TestProfile(RetriableStargateBridgeTest.Enabled.Profile.class)
-  class Enabled {
-    public static class Profile implements NoGlobalResourcesTestProfile {
-
-      @Override
-      public Map<String, String> getConfigOverrides() {
-        return ImmutableMap.<String, String>builder()
-            .put("stargate.grpc.retries.status-codes", "UNAVAILABLE,NOT_FOUND")
-            .put("stargate.grpc.retries.max-attempts", "2")
-            .build();
-      }
-    }
+  public static class Profile implements NoGlobalResourcesTestProfile {
 
-    @Test
-    public void notRetriedOnResponse() {
-      QueryOuterClass.Response response = QueryOuterClass.Response.newBuilder().build();
-
-      doAnswer(
-              invocationOnMock -> {
-                StreamObserver<QueryOuterClass.Response> observer = invocationOnMock.getArgument(1);
-                observer.onNext(response);
-                observer.onCompleted();
-                return null;
-              })
-          .when(bridgeService)
-          .executeQuery(any(), any());
-
-      QueryOuterClass.Query request = QueryOuterClass.Query.newBuilder().build();
-      bridge
-          .executeQuery(request)
-          .subscribe()
-          .withSubscriber(UniAssertSubscriber.create())
-          .awaitItem()
-          .assertItem(response)
-          .assertCompleted();
-
-      // verify one call only
-      verify(bridgeService).executeQuery(eq(request), any());
-    }
-
-    @Test
-    public void notRetriedWrongStatusCode() {
-      doAnswer(
-              invocationOnMock -> {
-                StreamObserver<QueryOuterClass.Response> observer = invocationOnMock.getArgument(1);
-                Status status = Status.UNIMPLEMENTED;
-                observer.onError(new StatusRuntimeException(status));
-                return null;
-              })
-          .when(bridgeService)
-          .executeQuery(any(), any());
-
-      QueryOuterClass.Query request = QueryOuterClass.Query.newBuilder().build();
-      Throwable result =
-          bridge
-              .executeQuery(request)
-              .subscribe()
-              .withSubscriber(UniAssertSubscriber.create())
-              .awaitFailure()
-              .getFailure();
-
-      assertThat(result)
-          .isInstanceOfSatisfying(
-              StatusRuntimeException.class,
-              e -> assertThat(e.getStatus()).isEqualTo(Status.UNIMPLEMENTED));
-
-      // verify one call only
-      verify(bridgeService).executeQuery(eq(request), any());
+    @Override
+    public Map<String, String> getConfigOverrides() {
+      return ImmutableMap.<String, String>builder()
+          .put("stargate.grpc.retries.status-codes", "UNAVAILABLE,NOT_FOUND")
```

Comment:
I guess this also tests that default policy is "status-codes". Could alternatively specify if explicitly here as documentation. Or maybe just add a comment it's the default (in case reader is wondering about it)

Reply:
adding a comment